src/fmu/dataio/datastructure/configuration/global_configuration.py

```diff
@@ -45,9 +45,7 @@ class Ssdl(BaseModel):
     """
 
     access_level: Optional[enums.Classification] = Field(default=None)
-    rep_include: Optional[bool] = Field(
-        default=False,
-    )
+    rep_include: Optional[bool] = Field(default=None)
```

Comment:
Changed the default here to None, to distinguish when it is present in the users config. And to avoid an incorrect UserWarning saying it should be removed from the config.
Should have been done in #694

Reply:
It should not be at the moment. The roundtrip we do on the config where we dump it back to dictionary while dropping `None` should then only trigger this 👇  part of the code if `rep_include` has an actual value in the config:

https://github.com/equinor/fmu-dataio/blob/827f29399f243fdbb1847f3a2075467b76193ff8/src/fmu/dataio/dataio.py#L525

But to be more future proof for when we address #629, I will change it to 
`if self.config.get("access", {}).get("ssdl", {}).get("rep_include") is not None:`

Actually when we have the config as a pydantic object we can use the `model_fields_set` property to check this instead 🙂